### PR TITLE
chore: replace heptio/contour with project contour

### PR DESCRIPTION
**Pull request by kushthedude**
Signed-off-by: kushthedude <kushthedude@gmail.com>
fixes https://github.com/projectcontour/contour/issues/2966

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -6,8 +6,8 @@ How you provide access to the service is going to be dependent on your specific
 
 A good starting point for yaml manifests for deploying to Kubernetes is in the [yaml](./yaml) directory.
 This creates a namespace, configmap, deployment and service.
-There is also an example ingress config that is set up to work with [Heptio Contour](https://github.com/heptio/contour), [JetStack cert-manager](https://github.com/jetstack/cert-manager) and [Let's Encrypt](https://letsencrypt.org/).
-See [Dave Cheney's blog post](https://blog.heptio.com/how-to-deploy-web-applications-on-kubernetes-with-heptio-contour-and-lets-encrypt-d58efbad9f56) for an example of getting all of that configured.
+There is also an example ingress config that is set up to work with [Project Contour](https://github.com/projectcontour/contour), [JetStack cert-manager](https://github.com/jetstack/cert-manager) and [Let's Encrypt](https://letsencrypt.org/).
+See [Project Contour's Guide](https://projectcontour.io/guides/cert-manager/) for help in getting started with Contour.
```
Comment (sunjayBhatia) [inline docs/README.md:10]:
```suggestion
See [this guide](https://projectcontour.io/guides/cert-manager/) for help getting started.
```

**Review by sunjayBhatia — CHANGES_REQUESTED** [review]
Looks good to me, just one wording change to make the sentences flow a bit better

**Review by youngnick — APPROVED** [review]
LGTM with @sunjayBhatia's requested change.

**Comment by kushthedude** [comment]
Hi @youngnick @sunjayBhatia I have did the requested changes, please review

**Review by sunjayBhatia — APPROVED** [review]

**Comment by sunjayBhatia** [comment]
Ah, I don't have official powers in this org, so someone else will have to approve officially

**Review by stevesloka — APPROVED** [review]
/lgtm 🎉

**Pull request merged**